hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/spi/impl/TestOzoneManagerServiceProviderImpl.java

```diff
@@ -201,8 +203,8 @@ public void testGetOzoneManagerDBSnapshot() throws Exception {
         .getOzoneManagerDBSnapshot();
     assertNotNull(checkpoint);
     assertTrue(checkpoint.getCheckpointLocation().toFile().isDirectory());
-    assertTrue(checkpoint.getCheckpointLocation().toFile()
-        .listFiles().length == 2);
+    assertEquals(2, Objects.requireNonNull(checkpoint.
```

Comment:
It's better to use 2 separate assertions here, AssertNotNull and AssertEquals.

Reply:
Thx @kaijchen  for the review. updated